nncf/common/tensor_statistics/statistics_serializer.py

```diff
@@ -71,6 +72,7 @@ def load_from_dir(dir_path: str) -> Tuple[Dict[str, Any], Dict[str, str]]:
             continue  # Skip the metadata file
 
         try:
+            fail_if_symlink(statistics_file)
```

Comment:
Could you add a specific function, such as `safe_open,` and use it here and below?